### DISCUSSION TITLE
Example browser error

### DISCUSSION THREAD

**Discussion by alabd14313** [bulletphysics/bullet3#3530] [General]
Hello!
In arch linux, I have this error when I run bullet3_examplebrowser.
`bullet3_examplebrowser: error while loading shared libraries: libBulletExampleBrowserLib.so.2.85: cannot open shared object file: No such file or directory
`

Here is it's packagebuild:
[PKGBUILD.txt](https://github.com/bulletphysics/bullet3/files/531628/PKGBUILD.txt)
Do you have any suggestion?

**Comment by erwincoumans** [comment]
The BulletExampleBrowser depends on libBulletExampleBrowserLib, so if you install it system-wide, you need to 'install' that library too, including various other libraries.
If you use the original Bullet premake or cmake build systems, it should all work fine.

**Comment by erwincoumans** [comment]
Could one also build/run that packagebuild install (PKGBUILD.txt) on Ubuntu, if so how?

**Comment by alabd14313** [comment]
@erwincoumans , here is some helps to create a PPA in ubuntu, though is not very straightforward:
https://wiki.debian.org/HowToPackageForDebian
http://askubuntu.com/questions/71510/how-do-i-create-a-ppa
